cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -99,8 +99,14 @@ if("@MSVC@")
   set(bootstrap_cmd "bootstrap.bat")
 elseif("@MINGW@")
   # Test scenario: MinGW installed in system, Visual Studio - not
-  set(install_cmd "b2")
-  set(bootstrap_cmd "bootstrap.bat" "gcc")
+  if("${CMAKE_CROSSCOMPILING}")
+    # cross compile from "linux" to "windows" using mingw
+    set(install_cmd "./b2")
```

Comment:
I think this one really depends on the host system, not on the toolchain. Probably it should be just:
```cmake
if(CMAKE_HOST_WIN32)
  set(install_cmd "b2")
  set(bootstrap_cmd "bootstrap.bat")
else()
  set(install_cmd "./b2")
  set(bootstrap_cmd "./bootstrap.sh")
endif()
```

See:
* https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_WIN32.html